controllers/users/verificationTokenUser.js

```diff
@@ -0,0 +1,36 @@
+import { User } from '#schemas/users.js'
+
+export const verificationTokenUser = async (req, res, next) => {
+  try {
+    const { verificationToken } = req.params
+    const user = await User.findOne({ verificationToken })
+
+    if (!user) {
+      return res.status(404).json({
+        status: '404 - Not Found',
+        'Content-Type': 'application/json',
+        ResponseBody: {
+          message: 'User not found',
+        },
+      })
+    }
+
+    const result = await User.findOneAndUpdate(
```

Comment:
A co jeśli użytkownik drugi raz wchodzi na ten link, teraz go ponownie dodajesz. Trzeba sprawdzić też czy verify is true.